SIG.md

```diff
@@ -0,0 +1,85 @@
+
+# SIG Roles
+
+## SIG Lead
+
+- Membership tracked in [sigs.yaml](./sigs.yaml)
+- MUST be a reviewer of one or more areas of the project, and should be a subject matter expert in their SIG area.
+- SHOULD drive charter changes (including creation) to get community buy-in but MAY delegate content creation to SIG contributors.
+- MUST organize a method of keeping cadence (meetings, async channel in Cilium Slack, etc.), and make sure [sigs.yaml](./sigs.yaml) is up to date.
+- SHOULD facilitate cadence but MAY delegate to other members or future Leads/Leads in training.
+- MUST ensure notes made available.
+- MAY delegate the actual communication and creation of content to other contributors where appropriate.
+
+### Requirements
+Leads MUST be at least a "reviewer" on our [contributor ladder](https://github.com/cilium/community/blob/main/CONTRIBUTOR-LADDER.md#reviewer) to be eligible to hold a leadership role within a SIG.
+SIGs MAY prefer various levels of domain knowledge depending on the role. This should be documented.
+
+### On-boarding and Off-boarding Leads
+Leads MAY decide to step down at anytime and propose a replacement. Use lazy consensus amongst the other Leads with fallback on majority vote to accept the proposal. The candidate SHOULD be supported by a majority of SIG contributors.
+Leads MAY select additional Leads through a super-majority vote amongst Leads. This SHOULD be supported by a majority of SIG contributors (active members of the Cilium organization engaged with creating and reviweing changes within the SIG).
+
+## Other Roles
+This governance document outlines the required roles for SIGs; however, SIGs are allowed to operate how they see fit outside of minimum governance requirements, including defining more roles to sustain the group. If a SIG needs to change the Lead position to include or remove duties, this needs to be approved by the committers. Newly created roles that don't assume any responsibility of Leads should follow the governing processes in the SIG's charter.
+
+# SIG Creation
+
+## Goals
+The primary goal of the charters is to define the scope of the SIG within Cilium and how the SIG leaders exercise ownership of these areas by taking care of their responsibilities. SIG Leads and SIGs are responsible for reviewing CFPs in their focus area, and have a responsibility to bring concerns to committers if they are unsure. 
+
+### SIG Charter Guide
+All Cilium SIGs must define a charter defining the scope and governance of the SIG.
+The scope must define what areas the SIG is responsible for directing and maintaining.
+The governance must outline the responsibilities within the SIG as well as the roles owning those responsibilities.
+
+### Steps to create a SIG charter
```

Comment:
Should there be a subsection mentioning how a SIG might be dissolved (other than by retirement)? Pessimistic example:

- Aspirant Lead open PR to propose new SIG
- Some Committers merge quickly
- Other Committers raise concerns
- Committers vote against the new SIG

Technically the new SIG has been created already. Similarly, if for some unlikely reason a SIG becomes counter-productive or in contradiction with the CoC, it might be necessary to dissolve it. Should there be a line to mention that the Committers can decide to dissolve a SIG under extraordinary circumstances?

Reply:
hmmm, I personally don't see the need to add a new section, but adding a sentence to the SIG retirement section to  outline a SIG may be dissolved if it doesn't abide by Cilium's CoC?

cc @xmulligan thoughts??

Reply:
Yeah doesn't have to be a full subsection, just a sentence is enough as long as it's mentioned somewhere, I'd say

Reply:
Yeah I think it should be clear that SIGs only exist at the direction of the Committers who ultimately decide everything in the project. Just as we are establishing SIGs now, we could dissolve them in future for a range of reasons.

Reply:
You phrased it much better than I did :)